lib/elixir/lib/kernel.ex

```diff
@@ -525,15 +526,16 @@ defmodule Kernel do
       :b
 
   """
-  @spec max(first, second) :: (first | second) when first: term, second: term
-  def max(first, second) do
-    :erlang.max(first, second)
+  @spec max(left, right) :: (left | right) when left: term, right: term
+  def max(left, right) do
+    :erlang.max(left, right)
```

Comment:
I can't quite put it into words but I really dislike `first` and `second` as they're terribly generic but at the same time they don't mention that they're generic "value"s or anything. But since I can't give a solid argument, let's push off this change for now and possibly revisit in the future.